From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Shelley Vohr <shelley.vohr@gmail.com>
Date: Mon, 9 Jan 2023 14:34:01 +0100
Subject: chore: remove unreached return

https://github.com/nodejs/node/pull/44376 added a return after a call to
ABORT(), triggering an unreachable code error.

This should be upstreamed.

diff --git a/src/node_builtins.cc b/src/node_builtins.cc
index 25b02f9904127780c68db9b451a7dfcf02a6fc5e..1be20aea500fd9d606c5a38a86b4aef66b49d133 100644
--- a/src/node_builtins.cc
+++ b/src/node_builtins.cc
@@ -247,7 +247,6 @@ void BuiltinLoader::AddExternalizedBuiltin(const char* id,
     fprintf(
         stderr, "Cannot load externalized builtin: \"%s:%s\".\n", id, filename);
     ABORT();
-    return;
   }
 
   icu::UnicodeString utf16 = icu::UnicodeString::fromUTF8(
